references/cli/cluster.go

```diff
@@ -223,7 +223,7 @@ func NewClusterJoinCommand(c *common.Args, ioStreams cmdutil.IOStreams) *cobra.C
 	}
 	cmd.Flags().StringP(FlagClusterName, "n", "", "Specify the cluster name. If empty, it will use the cluster name in config file. Default to be empty.")
 	cmd.Flags().StringP(FlagClusterManagementEngine, "t", multicluster.ClusterGateWayEngine, "Specify the cluster management engine. If empty, it will use cluster-gateway cluster management solution. Default to be empty.")
-	cmd.Flags().StringP(CreateNamespace, "", types.DefaultKubeVelaNS, "Specifies the namespace need to create in managedCluster")
```

Comment:
If empty, it will use default namespace `vela-system`.

Reply:
If --create-namespace option is given with empty string "" it won't create namespace. 

Please refer this [line](https://github.com/kubevela/kubevela/blob/73c9c75c6f1542147fdadcdf716025f889197ced/pkg/multicluster/cluster_management.go#L375)

```
 vela cluster join sample-cluster --name sample-cluster --create-namespace ""
 ```

Reply:
I understand. Please modify your PR title.

Reply:
Hi could you review?

Reply:
LGTM. ping @chivalryq